chef/cookbooks/crowbar-hacks/recipes/default.rb

```diff
@@ -91,7 +91,7 @@
 
   if neworder != node["uefi"]["boot"]["order"]
     Chef::Log.info("Change UEFI Boot Order: #{node[:provisioner_state]} #{node["uefi"]["boot"]["order"].inspect} => #{neworder.inspect}")
-    ::Kernel.system("efibootmgr -o #{neworder.map{ |e| sprintf("%x", order) }.join(",")}")
+    ::Kernel.system("efibootmgr -o #{neworder.map{ |e| sprintf("%x", e) }.join(",")}")
```

Comment:
Style/FormatString: Favor `format` over `sprintf`. (https://github.com/bbatsov/ruby-style-guide#sprintf)<br>Style/SpaceBeforeBlockBraces: Space missing to the left of {.